dataset_description.json

```diff
@@ -0,0 +1,5 @@
+{
+	"Name": "data_example_microscopy_sem",
+	"BIDSVersion":"1.4.0",

```

Comment:
@mariehbourget should we mention the BEP here? effectively, this dataset is _not_ compatible with BIDS 1.4.0

Reply:
You are absolutely right! We should add the version of the BEP which is: `Microscopy BEP031 version 0.0.2`.